File: public/phase-2-20160620022203/JAVASCRIPT/another-game-of-life.js
// Name : Another Game of Life
// input:
// n :  the size of the map
// m :  the number of populations that want to
var idx,exists,territory;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var m = parseInt(elements[1], 10);


// write your code here
// store your results in `exists`,`territory`

// Dummy Data
exists = "NO";
territory = [['']];


// output
print(exists);
if(exists=="YES"){
	for(idx=0;idx<n;idx++){
		print(territory[idx].join(' '));
	}
}

File: public/phase-2-20160620022203/PYTHON/another-game-of-life.py
#!/usr/bin/python
# Name : Another Game of Life
# input:
# n :  the size of the map
# m :  the number of populations that want to
elements = raw_input().strip().split(' ')
n = int(elements[0])
m = int(elements[1])


# write your code here
# store your results in `exists`,`territory`

# output

# Dummy Data
exists = "NO"
territory = [['']]

print(exists)
if(exists=="YES"):
	for idx in xrange(n):
		print(" ".join(str(e) for e in territory[idx]))